src/Header/Header.test.jsx

```diff
@@ -137,6 +137,28 @@ describe("Header", () => {
 		});
 	});
 
+	describe("Dropdown open and close callbacks", () => {
+		it("should call onDropdownOpen callback prop if dropdown is opened", () => {
```

Comment:
Is it worth adding tests for global functions, e.g. when passing in a string as the name of a function?